includes/controllers/class.llms.controller.certificates.php

```diff
@@ -117,10 +117,34 @@ public function maybe_handle_reporting_actions() {
 			$this->download( $cert_id );
 		} elseif ( isset( $_POST['llms_delete_cert'] ) ) {
 			$this->delete( $cert_id );
+		} elseif ( isset( $_POST['llms_change_shareable_cert'] ) ) {
+			$this->change_shareable_settings( $cert_id, isset( $_POST['llms_is_shareable_cert'] ) );
 		}
 
 	}
 
+	/**
+	 * Change shareable settings of a certificate
+	 *
+	 * @since 4.0.0
```

Comment:
Per doc standards: https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#using-placeholders

```suggestion
	 * @since [version]
```